docs/fastdds/api_reference/dds_pim/topic/contentfilteredtopic.rst

```diff
@@ -0,0 +1,10 @@
+.. _api_pim_contentfilteredtopic:
+
+.. rst-class:: api-ref
+
+ContentFilteredTopic
```

Comment:
Missing API is documented in eProsima/Fast-DDS#2585

Reply:
Tested against the branch